lego/apps/email/tasks.py

```diff
@@ -4,6 +4,7 @@
 from django.template.loader import render_to_string
 from django.utils import timezone
 
+import pytz
```

Comment:
Support for `pytz` was deprecated from [Django v4](https://docs.djangoproject.com/en/4.1/releases/4.0/#zoneinfo-default-timezone-implementation). Use `zoneinfo` instead.